packages/config-array/src/config-array.js

```diff
@@ -598,6 +586,11 @@ export class ConfigArray extends Array {
 		} else {
 			this.push(configs);
 		}
+
+		// On Windows, `path.relative()` returns an absolute path when given two paths on different drives.
+		// The namespaced base path is useful to make sure that calculated relative paths are always relative.
+		// On Unix, it is identical to the base path.
+		this.namespacedBasePath = toNamespacedPath(basePath || process.cwd());
```

Comment:
`process.cwd()` is always `/` in browser polyfills.

Reply:
I very much don't want to reference `process` in this package. It should not be thinking in terms of cwd because it may not be running in a server-side runtime.

Reply:
I would also prefer not to reference the current directory. The thing is that when `basePath` is an empty string or undefined, `path.relative()` resolves relative paths from the current directory. Do you think we can forbid `""` as a base path and always require that `basePath` is explicitly provided?